AllReadyApp/Web-App/AllReady/Areas/Admin/Views/Itinerary/Details.cshtml

```diff
@@ -125,6 +127,20 @@
         </div>
     </div>
 }
+<form method="post" action="@Url.Action("Details", "Itinerary", new { area="Admin" })#requests" class="form-horizontal">
```

Comment:
I have not tried adding a hash to a form action using the tag helper approach. If it doesn't work then adding `@Html.AntiForgeryToken()` should be sufficient here.